core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/parameter/network/AzureNetworkV4Parameters.java

```diff
@@ -81,6 +92,7 @@ public Map<String, Object> asMap() {
         putIfValueNotNull(map, "networkId", networkId);
         putIfValueNotNull(map, "subnetId", subnetId);
         putIfValueNotNull(map, "databasePrivateDsZoneId", databasePrivateDnsZoneId);
+        putIfValueNotNull(map, "aksPrivateDsZoneId", aksPrivateDnsZoneId);
```

Comment:
Where is the corresponding logic in `parse()`? Please use `AzureUtils.AKS_PRIVATE_DS_ZONE_ID` there as well instead of a hardwired String literal.